lib/snapshot/options.rb

```diff
@@ -47,6 +47,13 @@ def self.available_options
                                      default_value: [
                                        'en-US'
                                      ]),
+        FastlaneCore::ConfigItem.new(key: :launchArguments,
+                                     description: "A list of launch arguments which should be used",
+                                     is_string: false,
+                                     optional: true,
+                                     default_value: [
+                                      ''
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.